oomox_gui/colors_list.py

```diff
@@ -359,12 +420,14 @@ def __init__(self, display_name, key, callback, transient_for):
         self.color_entry = Gtk.Entry(
             text=_('<none>'), width_chars=6, max_length=6
         )
+        self.menu_button = OomoxLinkedDropdown(transient_for)
         self.color_entry.get_style_context().add_class(Gtk.STYLE_CLASS_MONOSPACE)
         linked_box = Gtk.Box(orientation=Gtk.Orientation.HORIZONTAL)
         Gtk.StyleContext.add_class(
             linked_box.get_style_context(), "linked"
         )
         linked_box.add(self.color_entry)
+        linked_box.add(self.menu_button)
```

Comment:
please swap places of menu_button and color_button (so the dropdown will be rightmost)